src/component/registry/bpmn-elements-registry.ts

```diff
@@ -133,63 +138,98 @@ export class BpmnElementsRegistry {
    * bpmnVisualization.bpmnElementsRegistry.addCssClasses('task_3', ['suspicious-path', 'additional-info']);
    * ```
    *
-   * **Notes**:
-   *
-   * - This method is intended to set CSS classes to specific elements, for instance to hide or highlight them. During BPMN diagram rendering, `bpmn-visualization` set specific CSS classes to all elements regarding their types.
-   * So, if you want to style all elements of a given type, use these default classes instead of adding new ones. The classes allow identifying elements of the same 'family' and of the same specific type.
-   * - For instance, a BPMN Service Task is an `Activity` and a `Task`, so it has the `bpmn-type-activity` and the `bpmn-type-task` classes. It shares these classes with all types of `Tasks`.
-   * It also has the specific `bpmn-service-task` to differentiate it from a BPMN User Task that has a `bpmn-user-task`.
-   * - In addition, labels also have the `bpmn-label` classes.
-   * - It is also possible to directly update the style of BPMN elements, see {@link updateStyle}.
-   * - Notice that if you pass ids that are not related to existing BPMN elements, their reference will be kept within the registry but nothing happens on the rendering side.
-   *
-   * See the repository providing the [examples of the `bpmn-visualization` TypeScript library](https://github.com/process-analytics/bpmn-visualization-examples/) for more details.
+   * @param bpmnElementIds The BPMN ID of the element(s) to add the CSS classes to.
+   * @param classNames The name of the class(es) to add to the BPMN element(s).
    *
-   * @param bpmnElementIds The BPMN id of the element(s) where to add the CSS classes
-   * @param classNames The name of the class(es) to add to the BPMN element(s)
+   * @see {@link removeCssClasses} to remove specific CSS classes from a BPMN element.
+   * @see {@link removeAllCssClasses} to remove all CSS classes from a BPMN element.
+   * @see {@link toggleCssClasses} to toggle CSS classes on a BPMN element.
+   * @see {@link updateStyle} to directly update the style of BPMN elements.
+   * @see The repository providing the [examples of the `bpmn-visualization` TypeScript library](https://github.com/process-analytics/bpmn-visualization-examples/) for more details.
```

Comment:
**nitpick:** Broken API doc in Typedoc. The following tags are not correctly displayed
- param
- see

![image](https://user-images.githubusercontent.com/27200110/236213571-2b6613b2-5b0c-4ef5-99b7-346d000ae7ad.png)